docker/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+mkdir -p /overlay/storage/{dokuwiki,work}
+mount -t overlay \
+      -o lowerdir=/overlay/original,upperdir=/overlay/storage/dokuwiki,workdir=/overlay/storage/work \
+         overlay /var/www/html
+
+docker-php-entrypoint apache2-foreground
```

Comment:
```suggestion
exec docker-php-entrypoint apache2-foreground
```

don't keep the process around, this also a common problem of signals not routed to pid=1